Best practices for Canvases and campaigns (#3836)
braze-inc/braze-docs: _docs/_user_guide/engagement_tools/campaigns/ideas_and_strategies/best_practices.md

---
nav_title: Best Practices
article_title: Campaign Best Practices
page_order: 0
description: "This article provides best practices for creating and customizing your campaigns."
tool: Campaign

---

# Campaign best practices

## User targeting

As you build out your campaigns over time, you may begin to notice lapses in your audience. At this crucial point, you can target your [lapsing users]({{site.baseurl}}/user_guide/engagement_tools/campaigns/ideas_and_strategies/capturing_lapsing_users/) with a specialized campaign using segmentation. 

### Identify your audience

Leverage segments and filters to your advantage by defining your audience. Consider who your campaign and messages are targeting. With this key information, you can create [multichannel campaigns]({{site.baseurl}}/user_guide/engagement_tools/campaigns/faq/#how-do-you-create-a-multichannel-campaign) that offer the flexibility of building your messages in different channels to match your audience's notification preferences.

> It's also important to understand your [active users]({{site.baseurl}}/user_guide/engagement_tools/campaigns/ideas_and_strategies/active_user_campaigns/) to show your appreciation to your consistent users.

Reviewer: > It's also important to understand your [active users]({{site.baseurl}}/user_guide/engagement_tools/campaigns/ideas_and_strategies/active_user_campaigns/) to show your appreciation to your consistent users.

This sentence is a tad bit confusing. What should they understand about their active users? Who they are? What they want?

## Multichannel campaigns

### Feature awareness

> If your goal is to draw your users towards a new feature or app version, be sure to use a multichannel strategy with a focus on in-app channels. [In-app messages][5] and [Content Cards][6] cards are generally less disruptive if a user doesn’t wish to update immediately. Be sure to include [deep links]({{site.baseurl}}/user_guide/personalization_and_dynamic_content/deep_linking_to_in-app_content/) to the appropriate app store.

Reviewer: ```suggestion
If your goal is to draw your users towards a new feature or app version, use a multichannel strategy with a focus on in-app channels. [In-app messages][5] and [Content Cards][6] are generally less disruptive if a user doesn’t wish to update immediately. Be sure to include [deep links]({{site.baseurl}}/user_guide/personalization_and_dynamic_content/deep_linking_to_in-app_content/) to the appropriate app store.
```

> Persuading a user to update their app or change the way they use your app can be difficult. So, make sure the let them know all the benefits of the new version or features and how it will improve their experience with your app. 

Reviewer: ```suggestion
Persuading a user to update their app or change the way they use your app can be difficult, so let them know about all the benefits of the new version or features and how it will improve their experience with your app. 
```

### Send timing

> Timing is key! When your goal is to convince users to update their app, wait until they have a positive experience within the app to ask users. To keep you audience engaged, avoid repetitive messaging that may appear intrusive.

Reviewer: ```suggestion
Timing is key! When your goal is to convince users to update their app, wait until they have a positive experience within the app to ask users. To keep your audience engaged, avoid repetitive messaging that may appear intrusive.
```

Over time, your users may forget certain features or not notice new features. When new features are added, be sure to let your users know with [in-app messages]({{site.baseurl}}/user_guide/message_building_by_channel/in-app_messages/about/). If users aren't engaging with major features within the app, it may be best to remind them when they're engaging with your app and when this new feature would be of use. Our article on [data opt-in][7] has more information on ways to ensure your request agrees with users' workflow expectations. 

## High ratings

Getting five-star ratings in the app store is on every mobile marketer's wish list. Achieving positive reviews, however, is no easy task because it requires extra work by your users. By applying Braze's functionality in clever ways, we can help you increase your customer engagement.

### Targeting power users

To ensure higher ratings, ask only your power users to review you in the app store. They are more likely to have great things to say. Although they differ from app to app, power users tend to have:

- Logged many sessions
- Used the app recently
- Spent money and made purchases

> For instance, you could create a segment named "Power users" with these filters:

Reviewer: ```suggestion
For example, you could create a segment named "Power users" with these filters:
```
